### ISSUE TITLE
Not relevant here, but...

### ISSUE THREAD

**Issue by pivic** [MicrosoftDocs/microsoft-style-guide#10]
There is a list entry for "SQL Server", but not for "Windows Server". Should "S" always be capitalised for "Windows Server" or not?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8708e1b8-fb2b-be8c-9df1-de8997168ae9
* Version Independent ID: be8ea9c3-663e-0a9c-f3af-704b5ea051be
* Content: [Windows, Windows-based - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/windows-windows-based)
* Content Source: [styleguide/a-z-word-list-term-collections/w/windows-windows-based.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/a-z-word-list-term-collections/w/windows-windows-based.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**

**Comment by camind** [comment]
Yes, the “S” is capitalized when you’re referring to the Windows Server operating system.
Don’t refer to the hardware that has Windows Server installed on it—that is, the server computer—as “a Windows Server,” though.
